### PR TITLE
[WIP] GAE test labels

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

File: appengine/php55/storage/test/LocalTest.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function testRead()
         $this->assertEquals($response->getContent(), $fileTxt);
     }
 
-    public function testWrite()
+    public function test_read_metadata_and_user_upload_and_write_caching_and_write_default_and_write_default_stream_and_write_metadata_and_write_options_and_write_public_and_write_simple_and_write_stream_should_testWrite()
```
Comment (ace-n) [inline appengine/php55/storage/test/LocalTest.php:70]: @bshaffer I think this was Takashi's concern 🙂

File: appengine/php55/http/test/LocalTest.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ public function testHome()
         $this->assertTrue($client->getResponse()->isOk());
     }
 
-    public function testFileRequest()
+    public function test_gae_issue_request_http_bin_should_testFileRequest()
```
Comment (bshaffer) [inline appengine/php55/http/test/LocalTest.php:44]: `public function testGaeIssueRequestHttpBin_shouldFileRequest()`?

File: appengine/php55/http/test/LocalTest.php [hunk 3]
```diff
@@ -53,7 +53,7 @@ public function testFileRequest()
         $this->assertResponse((string) $client->getResponse());
     }
 
-    public function testCurlRequest()
+    public function test_gae_issue_request_curl_request_should_testCurlRequest()
```
Comment (bshaffer) [inline appengine/php55/http/test/LocalTest.php:56]: `public function testGaeIssueRequestCurlRequest_shouldCurlRequest()`?

File: appengine/php55/twilio/test/LocalTest.php [hunk 4]
```diff
@@ -60,7 +60,7 @@ public function testSendText()
         $this->assertTrue($client->getResponse()->isOk());
     }
 
-    public function testTwiml()
+    public function test_twiml_should_testTwiml()
```
Comment (bshaffer) [inline appengine/php55/twilio/test/LocalTest.php:63]: `public function testTwilm_shouldTwiml()`?

File: appengine/php55/storage/test/LocalTest.php [hunk 5]
```diff
@@ -214,7 +214,7 @@ public function testServe()
         $this->assertContains('This is the contents of a file', CloudStorageTools::$served);
     }
 
-    public function testWritePublic()
+    public function test_write_public_should_testWritePublic()
```
Comment (bshaffer) [inline appengine/php55/storage/test/LocalTest.php:217]: `public function testWritePublic_shouldWritePublic()`?

File: appengine/php55/storage/test/LocalTest.php [hunk 6]
```diff
@@ -202,7 +202,7 @@ public function testWriteDefaultStream()
         $this->assertContains($time, $response->getContent());
     }
 
-    public function testServe()
+    public function test_image_serve_and_read_serve_should_testServe()
```
Comment (bshaffer) [inline appengine/php55/storage/test/LocalTest.php:205]: `public function testImageServe_ReadServe_shouldServe()`?

File: appengine/php55/storage/test/LocalTest.php (hunk 1 above)
Comment (bshaffer) [reply appengine/php55/storage/test/LocalTest.php:70]: how about: `testReadMetadata_UserUpload_WriteCaching_WriteDefault_WriteDefaultStream_WriteMetadata_WriteOptions_WritePublic_WriteSimple_WriteStream_shouldWrite()`

Not my favorite but if that's worst case scenario I'm fine with it

**Comment by bshaffer** [comment]
I love the new formatting! Eventually we can move the horribly-long-test-function-name case into annotations.

Nice work!

**Pull request merged**